src/pages/chat/Chat.tsx

```diff
@@ -129,7 +143,7 @@ const Chat = () => {
           </Allotment.Pane>
         )}
         <Allotment.Pane minSize={350}>
-          <Box sx={styles.chatContent(selectedChat)}>
+          <Box sx={styles.chatContent(!!selectedChat, messages.length > 0)}>
```

Comment:
same condition here, can be refactored to `message.length`

Reply:
fixed